src/state/actions/authAction.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { returnErrors } from './errorAction';
+
+
+import {
+  LOGIN_SUCCESS,
+  LOGIN_FAIL,
+} from './actionTypes';
+
+// Check token & load user
+export const login =({ email,password }) => dispatch => {
+
+ //request body
+
+ const body = JSON.stringify({ email,password })
+ console.log(email);
+ console.log(body);
+  axios
+    .post('https://phantom-api-pirates.herokuapp.com/users/login', {
+      email : email,
```

Comment:
Make this `URL` as an environment variable

Reply:
yeah sure